packages/wallets/src/features/cfd/screens/EnterPassword/EnterPassword.tsx

```diff
@@ -51,11 +51,9 @@ const EnterPassword: React.FC<TProps> = ({
     return (
         <div className='wallets-enter-password'>
             <div className='wallets-enter-password__container'>
-                {isDesktop && (
-                    <WalletText lineHeight='xl' weight='bold'>
-                        {modalTitle}
-                    </WalletText>
-                )}
+                <WalletText lineHeight='xl' weight='bold'>
+                    {modalTitle}
+                </WalletText>
```

Comment:
smol question: is this now needed for all devices?

Reply:
Yes based on the design and also for what we have in production this is currently being shown on all screen sizes.
![image](https://github.com/user-attachments/assets/5e15221e-4c01-4b15-b998-6aceb1493aee)
![image](https://github.com/user-attachments/assets/47ab3c6f-c29e-46ba-82d3-66c37943fd55)